ros/src/waypoint_updater/speed_calculator.py

```diff
@@ -17,60 +17,43 @@ def __init__(self, target_speed, current_speed=0.0,
                               target_acceleration, current_accleration,
                               acceleration_limit, jerk_limit)
         else:
-            # No acceleration. Current speed will be returned regardless of distance.
-            self.distance_speed_lut = [current_speed]
-
-        # The list of distances represented in the LUT
-        self.distances = [i * self.delta_d for i in range(len(self.distance_speed_lut))]
+            # No acceleration. Current speed/acceleration will be returned regardless of distance.
+            self.distances = [0.0]
+            self.velocities = [current_speed]
+            self.accelerations = [current_accleration]
```

Comment:
current_acceleration is not 0?

Reply:
Please check my answer to this question in #22, and continue any discussion there. This commit is only temporarily located in this pull-req until #22 is merged.